### PR TITLE
Adds repair command, updates theme

### LINKED ISSUE #13: [RELEASE] DirectPV 4.0.12
DirectPV 4.0.12 adds a new CLI command:

- [x] Add `kubectl directpv repair` to the doc [PR #880](https://github.com/minio/directpv/pull/880)

### PR DESCRIPTION
**Pull request by djwfyi**
Adds the `repair` command to the CLI docs.
Updates the theme.
Adds makefile and staging script.'

Closes #13 

Staged:
- [`repair` command](http://docs-nginx.minio.training:31080/directpv-docs/DOCS-13/command-line/repair/)
- [available command list](http://docs-nginx.minio.training:31080/directpv-docs/DOCS-13/command-line/#available-commands)

### CODE REVIEW

File: content/command-line/repair.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+tableOfContents: true
+---
+
+## Description
+
+{{< admonition title="Irrevocable Data Loss" type="warning" >}}
+This command completely and irreversibly erases any data that may exist on the selected drive(s).
+{{< /admonition >}}
+
+In a rare situation, the filesystem on a faulty XFS-formatted drive can be repaired to make them usable.
```
Comment (ravindk89) [inline content/command-line/repair.md:15]:
```suggestion
In the event of an XFS filesystem fault or corruption event, you can attempt a repair operation to restore the filesystem to usability. 
This operation has no guarantee of success and some or complete data loss is always a possibility.

Comment (ravindk89) [reply content/command-line/repair.md:15]: If this uses `xfs_repair` under the hood then:

```suggestion
DirectPV supports using the `xfs_repair` utility to attempt to repair managed drives that report XFS filesystem errors, faults, or corruption.
This command has no guarantee of success or complete recovery.
```

File: content/resource-management/drives.md [hunk 2]
```diff
@@ -161,6 +161,23 @@ $ kubectl directpv remove --drives=vdb --nodes=node1
 
 Refer to the [remove command]({{< relref "command-line/remove.md" >}}) for more information.
 
+## Repair Drives
+
+{{< admonition title="Data Loss" type="caution" >}}
+THIS IS DANGEROUS OPERATION WHICH LEADS TO DATA LOSS.
+{{< /admonition >}}
+
+In a rare situation, filesystem on a faulty XFS-formatted drive can be repaired to make it usable.
```
Comment (ravindk89) [inline content/resource-management/drives.md:170]: Same thought w.r.t. framing.

**Review by ravindk89 — APPROVED** [review]
LGTM mod suggestion on framing.

File: content/resource-management/drives.md (hunk 2 above)
Comment (djwfyi) [reply content/resource-management/drives.md:170]: It is, in fact, `xfs_repair` under the hood.
Updated to that suggestion in both places.

File: content/command-line/repair.md (hunk 1 above)
Comment (djwfyi) [reply content/command-line/repair.md:15]: Taking the `xfs_repair` suggestion.

**Pull request merged**